hivemind/client/beam_search.py

```diff
@@ -0,0 +1,275 @@
+import asyncio
```

Comment:
Note: this file contains a subset of DHT methods from 0.9.5. These methods now work on top of DHT.run_coroutine, but all algorithms are exactly the same.